src/xhydro/extreme_value_analysis/parameterestimation.py

```diff
@@ -283,14 +308,8 @@ def fit(
     out.attrs = prefix_attrs(
         ds.attrs, ["standard_name", "long_name", "units", "description"], "original_"
     )
-    dist = get_dist(dist)
-    attrs = dict(
-        long_name=f"{dist.name} parameters",
-        dist=dist.name,
-        method=METHOD_NAMES[method].capitalize(),
-        confidence_level=confidence_level,
-    )
-    out.attrs.update(attrs)
+
+    out.attrs.update(attrs_dist | dict(confidence_level=confidence_level))
```

Comment:
I don't think that you need to update the global attributes, since you already changed the variables. However, I would add something to retain the original attributes from `ds`.

`out.attrs = ds.attrs`